cabal.project

```diff
@@ -127,7 +127,7 @@ constraints:
   -- TODO: support hashable-1.4
   , hashable >=1.3.0.0 && <1.4
   , haskell-src-exts >=1.23.1 && <2
-  , hedgehog >=1.0.3 && <1.1
+  , hedgehog >=1.0.3 && <=1.1
```

Comment:
Why the departure from the usual <1.2? This case seems particularly like they must be equivalent, since this change explicitly adds support for 1.1.1

Reply:
Oh I didn't mean to commit this..